overlord/ifacestate/ifacemgr.go

```diff
@@ -51,6 +51,8 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, extraInterfaces
 		runner: runner,
 		repo:   interfaces.NewRepository(),
 	}
+	m.repo.SetSanitizePlugSlots()
```

Comment:
Same idea.. quite unexpected to have that initialized at a random point like this.